src/frontend/src/components/Header/MobileHeader.jsx

```diff
@@ -137,14 +170,25 @@ const MobileHeader = () => {
           >
             <MenuIcon className={classes.menuIcon} />
           </IconButton>
-          <Drawer
-            classes={{ paper: classes.paper }}
-            anchor="right"
-            open={state.right}
-            onClose={toggleDrawer('right', false)}
-          >
-            {sideList('right')}
-          </Drawer>
+          {username === false ? (
```

Comment:
Instead of `username === false ? ` flip the logic and use `username ?`  I don't think the username will be a pure boolean false, will it?  Rely on it being truthy instead.